app/assets/stylesheets/_board.scss

```diff
@@ -578,14 +578,40 @@
 
 .filters li a {
   position:relative;
-  padding: 10px;
+  padding: 4px;
+  font-size: 12px;
   display:block;
   padding-right:25px;
   text-decoration: none;
   border-radius: 4px;
   color:#777;
 }
 
+.filters {
+  .filter-group {
+    h5 {
+      &:hover {
+        background: #efefef;
+        border-color: #efefef;
+        border-style: solid;
+        border-radius: 2px;
+        border-width: 0px;
+      }
```

Comment:
Can we add a `cursor` style here?
<img width="152" alt="firefox_2016-07-28_13-52-44" src="https://cloud.githubusercontent.com/assets/133987/17225452/9d5bbd88-54ca-11e6-9dcb-00c53c3172c3.png">